lwallet/Command.hs

```diff
@@ -111,6 +113,20 @@ switch = lexeme $ positive $> True <|>
     positive = text "+" <|> text "y" <|> text "yes"
     negative = text "-" <|> text "n" <|> text "no"
 
+stakeholderId :: Parser StakeholderId
+stakeholderId = lexeme $ do
+    str <- many1 alphaNum
+    case decodeTextAddress (toText str) of
```

Comment:
This is wrong, why do you parse `StakeholderId` from `Address`? There is `Pos.Crypto.Hashing.decodeAbstractHash` to parse hash.
The main reason why it's wrong is that soon it will be impossible to extract `StakeholderId` from `Address`.